api/rbac/v1alpha1/role_types.go

```diff
@@ -27,13 +27,13 @@ type RoleResources struct {
 }
 
 type ResourceDetails struct {
-	ResourceType string   `json:"resourceType,omitempty"`
-	ResourceName string   `json:"resourceName,omitempty"`
-	Verbs        []string `json:"verbs,omitempty"`
+	ResourceType string   `json:"resourceType,omitempty" protobuf:"bytes,1,opt,name=resourceType"`
+	ResourceName string   `json:"resourceName,omitempty" protobuf:"bytes,2,opt,name=resourceName"`
+	Verbs        []string `json:"verbs,omitempty" protobuf:"bytes,3,rep,name=verbs"`
```

Comment:
I'm trying to figure out why these struct tags were absent before and have now shown up all of the sudden. 🤔

Reply:
This is non-blocking, btw.

Reply:
I was wondering the same thing. Maybe a library used in the tooling was updated recently, but this is the first PR since which had any codegen?